### DISCUSSION TITLE
Posible to make bot that get private message and save to file

### DISCUSSION THREAD

**Discussion by SlowSlinky** [ORelio/Minecraft-Console-Client#1892] [Q&A] [answered]
### Posible to make bot that get private message and save to file

Or console to show only private message

**Comment by SlowSlinky** [comment]
CLOSE

**Comment by ORelio** [comment, marked as answer]
I'd suggest [ChatLog](https://github.com/MCCTeam/Minecraft-Console-Client/blob/4ba09754de04a2d87fe921be6e7dd4b445edf2ad/MinecraftClient/Resources/config/MinecraftClient.ini#L140-L145). You may need to adjust [chat message detection](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#detecting-chat-messages).
